### ISSUE TITLE
Grimmia elatior : a new introduced moss species for Belgium

### ISSUE THREAD

**Issue by bramdhondt** [trias-project/unified-checklist#86]
As recently published in the yearly journal Muscillanea, the moss species _Grimmia elatior_ has been detected for the first time in Belgium, as an accidental introduction with rocks (Raeymaekers & De Beer 2023). Can this species be added to GRIIS, or will that happen through other datasets (e.g. Florabank)?

**Comment by timadriaens** [comment]
Hey, the 3 alien mosses so far (_Campylopus_, _Lophocolea_ and _Orthodontium lineare_) are in (my) [ad hoc checklist](https://www.gbif.org/dataset/1f3505cd-5d98-4e23-bd3b-ffe59d05d7c2) for now. Of course, the proper way would be to publish the entire moss (and plant) checklist, alien and native, properly. For now, I suggest to add the species to the ad hoc with all the necessary registry data. It was found on a rock in the Japanese garden so actually the status would probably still be "cultivated" (i.e. B2 - Individuals in cultivation (i.e. individuals provided with conditions suitable for them, but explicit measures to prevent dispersal are limited at best)? Can you add it please? You can find the file [here](https://docs.google.com/spreadsheets/d/1LeXXbry2ArK2rngsmFjz_xErwE1KwQ8ujtvHNmTVA6E/edit#gid=0).

**Comment by timadriaens** [comment]
Also suggest to move this issue to https://github.com/trias-project/ad-hoc-checklist

**Comment by bramdhondt** [comment]
@timadriaens : moved to https://github.com/trias-project/ad-hoc-checklist/issues/43 , so I assume this Issue can be deleted.

**Comment by qgroom** [comment]
I don't think "cultivated" fits what you describe. 
I wonder if released (category B3) in the Degree of Establishment controlled vocabulary https://dwc.tdwg.org/doe/#dwcdoe_d004
would be a better fit.

**Comment by timadriaens** [comment]
Indeed, released (or even casual) would fit better, tx. I will put it on released.
